pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1507,6 +1507,14 @@ self: super: {
   # Fixed at head, but hasn't cut a release in awhile.
   darcs = doJailbreak super.darcs;
 
+  # Test suite's dependencies (QuickCheck etc.) are outdated
+  # see https://github.com/kowainik/typerep-map/issues/81
```

Comment:
This issue was resolved. New [`typerep-map-0.3.3.0`](https://hackage.haskell.org/package/typerep-map-0.3.3.0) has been released on Hackage.